src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -247,6 +251,9 @@ HARDWARE_INTRINSIC(Sve,           StoreAndZipx2,
 HARDWARE_INTRINSIC(Sve,           StoreAndZipx3,                                                    -1,      3,      true,  {INS_sve_st3b,       INS_sve_st3b,       INS_sve_st3h,       INS_sve_st3h,       INS_sve_st3w,       INS_sve_st3w,       INS_sve_st3d,       INS_sve_st3d,       INS_sve_st3w,       INS_sve_st3d},    HW_Category_MemoryStore,           HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_NeedsConsecutiveRegisters)
 HARDWARE_INTRINSIC(Sve,           StoreAndZipx4,                                                    -1,      3,      true,  {INS_sve_st4b,       INS_sve_st4b,       INS_sve_st4h,       INS_sve_st4h,       INS_sve_st4w,       INS_sve_st4w,       INS_sve_st4d,       INS_sve_st4d,       INS_sve_st4w,       INS_sve_st4d},    HW_Category_MemoryStore,           HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_NeedsConsecutiveRegisters)
 
+HARDWARE_INTRINSIC(Sve,           ConditionalExtractAfterLastActiveElementScalar,                    0,      3,      false, {INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta},  HW_Category_Scalar,                HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_HasRMWSemantics|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation)
```

Comment:
Please place these in  alphatical order.

Reply:
With what exactly? `SaturatingDecrementBy*Scalar` that follow the three added lines are out of alphabetical order with `StoreAndZipx*` above. Are you asking to put these next to other non-scalar `ConditionalExtract*`?

Reply:
Everything under `// Special intrinsics that are generated during importing or lowering` should be alphabetically ordered. We may have missed doing it for `SaturatingDecrementBy*`. Do you mind fixing that as well?

Reply:
Done.